app/controllers/onboarding_controller.rb

```diff
@@ -17,28 +18,43 @@ def create
     return redirect_to_success if User.email_taken?(user_params[:email])
 
     @user = User.new(user_params)
-    return redirect_to_success if @user.save
+    if @user.save
+      JsonWebToken.create(invalidated_jti: params[:jwt])
+      return redirect_to_success
+    end
 
     render :index
   end
 
   def success; end
 
+  def create_invite_url
+    payload = SecureRandom.base64(16)
+    jwt = JsonWebToken.encode(payload)
+    render json: { url: URI::HTTPS.build(path: '/onboarding', query: "jwt=#{jwt}") }
```

Comment:
When I run this locally, I get an URL like `https:/onboarding?jwt=abc`. I think I’d make use of Rails’ url helpers here which should also make sure that we use the correct protocol scheme and host name.

Something like this should work:

```ruby
onboarding_path(jwt: jwt)
```

Reply:
way simpler hehe